modules/cactus-web/src/AccessibleField/AccessibleField.tsx

```diff
@@ -139,7 +144,7 @@ function AccessibleFieldBase(props: AccessibleFieldProps): React.ReactElement {
           label={props.tooltip}
           id={tooltipId}
           maxWidth={maxWidth}
-          disabled={disabled}
+          disabled={disableTooltip}
```

Comment:
As you have it, when `disableTooltip` is `undefined`, then the tooltip will never be disabled.  The third AC states that we should fall back to the current behavior if it's `undefined`.  Instead, I'd change this to:

```suggestion
          disabled={disableTooltip ?? disabled}
```